### PR TITLE
Dashboard list item#14

### LINKED ISSUE #14: Move dashboard list item component from the clark-client into the admin dashboard
The list item component from the dashboard should be used in the admin dashboard to replace the dashboard rows. This will entail moving the clark-contextmenu plugin over as well and integrating it with the existing context menu.

### PR DESCRIPTION
**Pull request by jbarlow96**
Move list-item component from clark client to admin dashboard.
Add an icon for the 'proofing' tooltip.

closes #14

### CODE REVIEW

File: src/_vars.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@ $error-red: #cf4739;
 $placeholder: lighten(#a9a9a9, 10);
 $popup-z: 9;
 $notification-z: 10;
+$cube-pattern: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1NiIgaGVpZ2h0PSIxMDAiPgoKPHBhdGggZD0iTTI4IDY2TDAgNTBMMCAxNkwyOCAwTDU2IDE2TDU2IDUwTDI4IDY2TDI4IDEwMCIgZmlsbD0ibm9uZSIgc3Ryb2tlPSJ3aGl0ZSIgc3Ryb2tlLXdpZHRoPSIyIj48L3BhdGg+CjxwYXRoIGQ9Ik0yOCAwTDI4IDM0TDAgNTBMMCA4NEwyOCAxMDBMNTYgODRMNTYgNTBMMjggMzQiIGZpbGw9Im5vbmUiIHN0cm9rZT0id2hpdGUiIHN0cm9rZS13aWR0aD0iMiI+PC9wYXRoPgo8L3N2Zz4=');
```
Comment (gshaw1997) [inline src/_vars.scss]: Is this style actually being used?

Comment (jbarlow96) [reply src/_vars.scss]: It needed to be added or the dashboard wouldn't be compiled.

File: src/app/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -8,7 +19,170 @@ import { FilterBarService } from 'app/core/filter-bar.service';
...
+          new LearningObject(object)
+        );
+        const total = response.total;
+        return { objects, total };
+      });
+  }
+
+  viewUser(object: LearningObject): void {
+    this.filterSearchService.setSearchText(object.status);
+    this.router.navigate(['/dashboard'], { skipLocationChange: true });
```
Comment (nrwinner) [inline src/app/dashboard/dashboard.component.ts]: Why are we skipping location change here? Why not just expose the routes?

File: src/app/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -1,5 +1,16 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild, ChangeDetectorRef } from '@angular/core';
```
Comment (nrwinner) [inline src/app/dashboard/dashboard.component.ts]: Why is ChangeDetectorRef imported and injected into this component?

File: src/app/dashboard/learning-object-management/learning-object-management.component.html [hunk 4]
```diff
@@ -19,10 +22,9 @@
       Last Updated
     </div>
   </div>
-  <div class="grid-row learning-object" [ngClass]="{published: object.published, unpublished: !object.published && !object.lock, locked: object.lock, attention: +object.lock?.date < +object.date}"
+  <!-- <div class="grid-row learning-object" [ngClass]="{published: object.published, unpublished: !object.published && !object.lock, locked: object.lock, attention: +object.lock?.date < +object.date}"
```
Comment (DonSeannelly) [inline src/app/dashboard/learning-object-management/learning-object-management.component.html]: Commented code should be removed

File: src/app/dashboard/dashboard.component.ts [hunk 5]
```diff
@@ -8,7 +19,170 @@ import { FilterBarService } from 'app/core/filter-bar.service';
...
+    private cd: ChangeDetectorRef,
+    private contextMenuService: ContextMenuService,
+    private auth: AuthService,
+    private filterSearchService: FilterSearchService,
+    private router: Router,
+    private http: Http
+  ) {
+    this.headers.append('Content-Type', 'application/json');
+  }
+  NgOnInit() {
```
Comment (nvisal1) [inline src/app/dashboard/dashboard.component.ts]: NgOnInit is a lifecycle hook for components. This will not run within a service

**Comment by jbarlow96** [comment]
screen shot 2019-02-07 at 11 59 02 am

**Comment by jbarlow96** [comment]
screen shot 2019-02-07 at 11 59 22 am
screen shot 2019-02-07 at 11 59 38 am

**Comment by nrwinner** [comment]
@jbarlow96 Can we establish a new color for the proofing stage? Love the icon selection. Right now it's orange like review though and I think I'd like to see it differentiated a little bit.

**Comment by jbarlow96** [comment]
screen shot 2019-02-07 at 4 19 42 pm

**Comment by jbarlow96** [comment]
screen shot 2019-02-07 at 4 28 43 pm

**Pull request merged**